kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/DataServiceImpl.java

```diff
@@ -913,13 +913,13 @@ public void run() {
                 }
                 connected = true;
             } catch (KuraConnectException | KuraStoreException e) {
-                logger.warn("Connect failed", e);
-
+                logger.warn("DataService failure occured: ", e);
```

Comment:
Maybe we can also add the exception class name as part of the message since it is not easily visible if the user looks at systemd journal, something like `logger.warn("Connection attempt failed: {}", e.getClass().getSimpleName(), e);`